.github/workflows/translation-test.yml

```diff
@@ -0,0 +1,30 @@
+name: Translation Test
+
+on: push
```

Comment:
Is this action currently running on any push to a branch? I'm wondering if we can be more selective about when this action is run. We definitely want to make sure we don't merge anything into master that's missing translations. But, it's possible that we merge some code into `develop`, say, that doesn't have translations yet. We may have finished the feature but are waiting on Smartling for the translations.

Reply:
updated to only run on release branches and master!